src/snippets/lazy-load-image.liquid

```diff
@@ -0,0 +1,31 @@
+{% assign widthsArray = imageWidths | split: " " %}
```

Comment:
This is cool, but are we reinventing the implementation?  https://www.shopify.com/partners/blog/using-responsive-images is what I want us to be moving toward. Do we need this additional snippet?